packages/peregrine/lib/Apollo/policies/index.js

```diff
@@ -170,7 +170,7 @@ const typePolicies = {
     SelectedConfigurableOption: {
         // id alone is not enough to identify a selected option as it can refer
         // to something like "size" where value_id refers to "large".
-        keyFields: ['id', 'value_id']
+        keyFields: ['uid', 'value_id']
```

Comment:
In addition to this, we should also add the following type policy too so Apollo can correctly cache the ConfigurableProductOptions type:

```
ConfigurableProductOptions: {
        keyFields: ['uid']
},
```

It should show up in the cache as:

```
ConfigurableProductOptions:{"uid":"Y29uZmlndXJhYmxlLzExMzkvMTc5"}
```

Instead of

```
ConfigurableProductOptions:120
```